plugins/webhooks/github.js

```diff
@@ -173,8 +175,8 @@ function pullRequestEvent(request, reply) {
     const prNumber = hoek.reach(payload, 'pull_request.number');
     const repository = hoek.reach(payload, 'pull_request.base.repo.ssh_url');
     const branch = hoek.reach(payload, 'pull_request.base.ref');
-    const scmUrl = `${repository}#${branch}`;
-    const prRef = `${repository}#pull/${prNumber}/merge`;
+    const scmUrl = scmGithub.formatScmUrl(`${repository}#${branch}`);
```

Comment:
I agree, instantiating an object just for this case is awkward, thats why a better solutions is to make a static function or use the scmPlugin that's exposed in the server

I feel like it's definitely the right solution to be using the `formatScmUrl` over just lower-casing the `repository` though